_includes/head.html

```diff
@@ -90,7 +90,7 @@
 // export some helpful site-wide variables
 window.site = {
   baseurl: "{{ site.baseurl }}"{% unless site.public %},
-  access_token: "<!--# echo var="access_token" -->"{% endunless %}
+  access_token: '<!--# echo var="access_token" -->'{% endunless %}
```

Comment:
Using double quotes, the string is broken by the double quotes are `"access_token"`. This makes the JS invalid and so it fails linting.

Also, when running the hub locally without nginx, this server side include doesn't work, so the broken string causes a JS error that blocks the rest of the script.

By switching to single quotes, everything stays valid. If there SSI doesn't get replaced, the string is just sent as the token, which is invalid, and midas authentication fails (which is expected and OK behavior).